lib/python/picongpu/extra/input/preparingInsightData_example.ipynb

```diff
@@ -39,12 +39,12 @@
    "metadata": {},
    "source": [
     "## Get the data\n",
-    "The far field data from an Insight measurement is stored in a h5 file and typically measured in dependence of two transversal coordinates (\"x\" and \"y\" in mm) and the frequency (\"w\" in rad/fs). If there are any deviations from this scheme, you will have to adjust the _PrepRoutines_ source code. \n",
+    "The far field data (= at the focal plane) from an Insight measurement is stored in a h5 file and typically measured in dependence of two transversal coordinates (\"x\" and \"y\" in mm) and the frequency (\"w\" in rad/fs). If there are any deviations from this scheme, you will have to adjust unit of the speed of light `c` in line 32 of the _PrepRoutines_ source code. \n",
     "\n",
     "Then, the first steps are:\n",
     "1. read the far field data and store it in numpy arrays\n",
     "2. fit the far field data intensity with a 2D gaussian to extract the beam center and waist size\n",
-    "3. propagate to the near field\n",
+    "3. propagate to the near field (= just before the the diffracting element)\n",
```

Comment:
```suggestion
    "3. propagate to the near field (= just before the diffracting element)\n",
```

Reply:
I tried it with "ci: no-compile", since those changes will not affect PIConGPU at all; but either I made something wrong or it would not let me, and the CI compiled fully. Sorry about that.
And GVD and GDD are physically different things. GVD is a material constant, and a pulse running through media with GVD will have GDD in the end. Since the material is not important here but only the result in the end, GDD is the right choice.